pontoon/base/utils.py

```diff
@@ -643,7 +643,7 @@ def build_translation_memory_file(creation_date, locale_code, entries):
         }
     )
     for resource_path, key, source, target, project_name, project_slug in entries:
-        tuid = ':'.join((project_slug, slugify(resource_path), slugify(key)))
+        tuid = ':'.join((project_slug, resource_path, slugify(key)))
```

Comment:
Outside the scope of this patch, but small enough.

This change was introduced as part of bug 1390149. We don't need to slugify resource path, so let's keep it usable.